src/bikeracks/votes.py

```diff
@@ -69,37 +69,17 @@ def submit_vote():
     # connect to db
     db = get_db()
     
-    vote_status = h.get_vote_status(db, rack_id, user_id)
-    if vote_status:
-        # updating an existing vote
-        
-        query = "UPDATE votes SET vote_type = ? WHERE rack_id = ? AND user_id = ?"
-        db.execute(query, (vote_type, rack_id, user_id))
-        db.commit()
-        # also update the vote count in bikeracks table
-        if vote_type == 1:
-            # update the downvote_count in bikeracks for this rack, decrement by 1
-            h.decrement_downvote_count(rack_id, db)
-            
-        # if the user downvoted
-        elif vote_type == -1:
-            # update the upvote_count in bikeracks for this rack, decrement by 1
-            h.decrement_upvote_count(rack_id, db)
-        
-    else:
-        # voting for the first time
-        
-        query = "INSERT INTO votes (rack_id, user_id, vote_type) VALUES (?, ?, ?)"
-        db.execute(query, (rack_id, user_id, vote_type))
-        db.commit()
-        
-    # increment the upvote or downvote count
+    query = """ INSERT INTO votes (rack_id, user_id, vote_type)
+                VALUES (?, ?, ?)
+                ON CONFLICT(rack_id, user_id) DO UPDATE SET vote_type=?"""
+    db.execute(query, (rack_id, user_id, vote_type))
```

Comment:
Did you test this? I think you're missing an argument.

Reply:
I tested it in command line for the reason explained in previous comment. Added the missing argument.

Reply:
How did it pass your command line testing despite having the wrong number of arguments?

Reply:
I added the missing argument.